src/cascadia/TerminalCore/pch.h

```diff
@@ -10,4 +10,13 @@
 #include "winrt/Windows.Foundation.h"
 
 #include "winrt/Microsoft.Terminal.Core.h"
+
+// Including TraceLogging essentials for the binary
+#include <TraceLoggingProvider.h>
+#include <winmeta.h>
+// We'll just use the one in TerminalControl, since that's what we'll get embedded in.
+TRACELOGGING_DECLARE_PROVIDER(g_hTerminalControlProvider);

```

Comment:
_how does this work for PublicTerminalCore_? does it even build? can you make sure that works? What about Visual Studio..?

Reply:
fixed this